Cabal/Distribution/PackageDescription/FieldGrammar.hs

```diff
@@ -539,3 +542,16 @@ setupBInfoFieldGrammar def = flip SetupBuildInfo def
     <$> monoidalFieldAla "setup-depends" (alaList CommaVCat) L.setupDepends
 {-# SPECIALIZE setupBInfoFieldGrammar :: Bool -> ParsecFieldGrammar' SetupBuildInfo #-}
 {-# SPECIALIZE setupBInfoFieldGrammar :: Bool ->PrettyFieldGrammar' SetupBuildInfo #-}
+
+-------------------------------------------------------------------------------
+-- Script
+-------------------------------------------------------------------------------
+
+scriptFieldGrammar
+    :: (FieldGrammar g, Applicative (g Script), Applicative (g Executable), Applicative (g BuildInfo))
+    => g Script Script
+scriptFieldGrammar = Script
+    <$> blurFieldGrammar L.executable (executableFieldGrammar "script")
+    <*> optionalFieldAla "with-compiler" FilePathNT L.hcPath
+{-# SPECIALIZE scriptFieldGrammar :: ParsecFieldGrammar' Script #-}
+{-# SPECIALIZE scriptFieldGrammar :: PrettyFieldGrammar' Script #-}
```

Comment:
There's no EOL newline. Not biggie, but if you can configure your editor for this project, it would be nice next time!